packages/task/src/browser/task-configurations.ts

```diff
@@ -146,19 +146,16 @@ export class TaskConfigurations implements Disposable {
     }
 
     getRawTaskConfigurations(scope?: TaskConfigurationScope): (TaskCustomization | TaskConfiguration)[] {
-        if (!scope) {
+        if (scope === undefined) {
```

Comment:
Since the enum has the non-string scopes as `1` and `2`, this wasn't causing an issue, but it could have if a scope ended up with enum value `0`.

Reply:
Looks good 👍